.typos.toml

```diff
@@ -9,6 +9,9 @@ extend-ignore-identifiers-re = [
     "NOOP",
     "Nam",
     "typ",
+    "ALOC",
```

Comment:
This has been fixed already, I think you might have a conflict when branch is rebased on Github.

Reply:
Nice. Will rebase them.